packages/asset-swapper/src/types.ts

```diff
@@ -14,10 +14,8 @@ import {
 } from './utils/market_operation_utils/types';
 import { QuoteReport } from './utils/quote_report_generator';
 
-export interface APIOrder extends Omit<APIOrderV3, 'order'> {
-    order: LimitOrderFields;
-    signature: Signature;
-}
+// TODO (Xianny): move to @0x/types
+export type SignedLimitOrder = LimitOrderFields & { signature: Signature };
```

Comment:
nit: can we use the `Signature` exported from `protocol-utils`? To avoid having these types diverge